### ISSUE TITLE
Fraud Detection with Logistic Regression

### ISSUE THREAD

**Issue by akshaydubey05** [Akshat111111/Hedging-of-Financial-Derivatives#91]
**Objective** : This model would predict whether a transaction is fraudulent or not based on transaction features. 

This model would **understand** data, preprocess, train logistic regression; assess metrics, interpret coefficients; deploy for **real**-**time** **security**

**Comment by akshaydubey05** [comment]
Hi, i created this issue, please assign it to me under GSSOC'24.

**Comment by RishiRaj292** [comment]
Hello @Akshat111111 , could you please also add the label 'status' to show whether the issue has been assigned or not to a person (And preferably to all the issues if possible)? This would save a lot of time for the others viewing the project for issues.
Thank you

**Comment by Saswatsusmoy** [comment]
@Akshat111111 Instead of using logistic regression, better ML models which can capture intricate features and help in better classification can be used. Decision Trees and Random Forest are better options because they help in preventing overfitting in these cases which is a common problem in financial fraud detection models.


It would be better if you can change the issue to detect fraudulent transaction instead of specifying a single ML algorithm

**Comment by akshaydubey05** [comment]
Hi @Saswatsusmoy thanks for the suggestion ,I thought of making this issue using **Random Forest** as it is the most efficient one, but at that time i was learning it. No woories now i will do this task with both **Linear Regression** and **Random Forest**.

**Comment by Akshat111111** [comment]
> Hello @Akshat111111 , could you please also add the label 'status' to show whether the issue has been assigned or not to a person (And preferably to all the issues if possible)? This would save a lot of time for the others viewing the project for issues. Thank you

I will advice you to create a new issue as currently all the issues are assigned and the peoples are working on it.

**Comment by Srijansarkar17** [comment]
Hey @Akshat111111 I would like to solve this issue. Can you please assign it to me? I will use neural networks for this problem. Adding hidden layers will improve the accuracy too.

**Comment by Akshat111111** [comment]
> Hey @Akshat111111 I would like to solve this issue. Can you please assign it to me? I will use neural networks for this problem. Adding hidden layers will improve the accuracy too.

I will suggest you to create a new issue and tell your approach in a brief way .

**Comment by Samik123Mit** [comment]
pls assign me the issue,i have prior experience with the model task.
